proquotes/views/Quote/quotesFrontend.xml

```diff
@@ -54,64 +56,104 @@
-				<t t-if="line.name == '$hardware'">
-					<tr>
-						<td style="border-style: none;" colspan="100">
-							<span id="lang-id" class="english-mode no-display">english-mode</span>
-							<t t-call="proquotes.renewal-hardware-english" />
-							<t t-call="proquotes.renewal-hardware-french" />
-						</td>
-					</tr>
-				</t>
-				<t t-if="line.name == '$software'">
-					<tr>
-						<td style="border-style: none;" colspan="100">
-							<t t-call="proquotes.renewal-software" />
-						</td>
-					</tr>
-				</t>
-				<t t-if="line.name == '$subscription'">
-					<tr>
-						<td style="border-style: none;" colspan="100">
-							<t t-call="proquotes.renewal-subscription" />
-						</td>
-					</tr>
-				</t>
-				<tr class="quote-head" t-if="line.name == '$block'">
+			<t t-if="line.name == '$hardware'">
+				<tr>
+					<td style="border-style: none;" colspan="100">
+						<span id="lang-id" class="english-mode no-display">english-mode</span>
+						<t t-call="proquotes.renewal-hardware-english" />
+						<t t-call="proquotes.renewal-hardware-french" />
+					</td>
+				</tr>
+			</t>
+			<t t-if="line.name == '$software'">
+				<tr>
+					<td style="border-style: none;" colspan="100">
+						<t t-call="proquotes.renewal-software" />
+					</td>
+				</tr>
+			</t>
+			<t t-if="line.name == '$subscription'">
+				<tr>
+					<td style="border-style: none;" colspan="100">
+						<t t-call="proquotes.renewal-subscription" />
+					</td>
+				</tr>
+			</t>
+			<t t-if="line.name == '$rental_pricenote'">
+				<tr>
+					<td style="border-style: none;" colspan="100">
+						<p class="rental_price_explain" t-if="clang == 'en_CA' or clang == 'en_US'">
+							* The final rental cost will be calculated once the equipement is returned.
+							Weekly rental cost is equivalent to 4x the indicated daily rental rate (above).
+							Monthly rental cost is equivalent to 12x the indicated daily rental rate (above). ‌
+						</p>
+						<p class="rental_price_explain" t-elif="clang == 'fr_CA'">
+							* Le coût final sera calculé lorsque l'équipement sera retourné.
+							Le taux de location à la semaine est équivalent à 4x le taux journalier ci-haut.
+							Le taux de location au mois est équivalent à 12x le taux journalier ci-haut.
+						</p>
+						<br />
+						<div class="rental_product_value" t-if="clang == 'en_CA' or clang == 'en_US'">
+							<span> Total Rental Value:&#160;<span id="total-rental-value"></span>
+							</span>
+							<br />
+							<span>
+								Please upload your proof of insurance for the total amount in the comments below:
+							</span>
+							<br />
+						</div>
+						<div class="rental_product_value" t-if="clang == 'fr_CA'">
+							<span>Valeur locative totale: &#160;<span id="total-rental-value"></span>
+							</span>
+							<br />
+							<span>
+								Veuillez télécharger votre preuve d'assurance pour le montant total dans les commentaires ci-dessous:
```

Comment:
ci-dessous: 
should be
ci-dessous : 

In french, we need to put a space before writing the colon